src/components/RequestBlock/index.tsx

```diff
@@ -174,7 +174,7 @@ const RequestBlock: React.FC<RequestBlockProps> = ({ request, onUpdate }) => {
                 disabled={isUpdating}
               >
                 <svg
-                  className="w-4 h-4"
+                  className="w-6 h-6"
```

Comment:
All the icon size changes in this file should be reverted.

Reply:
Yeah sorry, those were a mistake from when I was replacing the checkbox sizes!  Combing through to find them all now.